run.py

```diff
@@ -72,7 +72,7 @@ def on_app_mention(event_data):
             return
 
         cursor.execute(
-            "DELETE FROM slack_client_msg_id WHERE created_at < CURRENT_TIMESTAMP - '00:10:00'",
+            "DELETE FROM slack_client_msg_id WHERE created_at < CURRENT_TIMESTAMP - interval '10 minutes'",
```

Comment:
こうすると良さそう？

```suggestion
            "DELETE FROM slack_client_msg_id"
            "WHERE created_at < CURRENT_TIMESTAMP - interval '10 minutes'",
```

https://github.com/dev-hato/hato-bot/pull/624#issuecomment-862902143

```
************* Module run
run.py:75:0: C0301: Line too long (107/100) (line-too-long)

-----------------------------------
Your code has been rated at 9.99/10
```